Desktop/modules/upgrader/upgrade.cpp

```diff
@@ -66,7 +69,7 @@ Version Upgrade::fromVersion() const
 	}
 	catch(Version::encodingError & e)
 	{
-		throw upgradeError("Incorrect from version: '" + fq(_fromVersion) + "' supplied to Upgrade...");
+		throw upgradeError("Incorrect from-version: '" + fq(_fromVersion) + "' supplied to Upgrade...");
```

Comment:
```suggestion
		throw upgradeError("Incorrect fromVersion: '" + fq(_fromVersion) + "' supplied to Upgrade...");
```
If this should match the properties in `Upgrader.qml`, I would use their exact names.